CMakeLists.txt

```diff
@@ -187,6 +187,10 @@ option(IRR_BUILD_DOCS "Enable building documentation?" ON)
 
 set(THIRD_PARTY_SOURCE_DIR "${PROJECT_SOURCE_DIR}/3rdparty")
 
+unset(OPENEXR_BUILD_UTILS [OPENEXR]) #disable openexr examples
+unset(BUILD_TESTING [BUILD]) #disable openexr bulding tests
+unset(PYILMBASE_ENABLE [PYILMBASE]) #disable PYILMBASE in openexr
```

Comment:
are you sure this works? To unset a thing I guess it would have to exist first, but you also need to set it before the subfolder is included.

I usually just set the thing to false, right before the subfolder include.

Reply:
I'm not sure being honest. Well, I will do as you wrote, but I've not known how to do it yet. To fix.

Reply:
put this in `3rdparty/CMakeLists.txt`